cpp/include/cudf/binaryop.hpp

```diff
@@ -53,14 +53,12 @@ enum class binary_operator : int32_t {
                    ///< ptx code
   SHIFT_LEFT,      ///< operator <<
   SHIFT_RIGHT,     ///< operator >>
-
   // Logical right shift. Casts to an unsigned value before shifing.
   // approximates >>> from Java.
   SHIFT_RIGHT_UNSIGNED,  ///< operator >>>
-
-  LOG_BASE,  ///< logarithm to the base
-
-  INVALID_BINARY  ///< invalid operation
+  LOG_BASE,              ///< logarithm to the base
+  ATAN2,                 ///< atan2 operator
```

Comment:
More meaningful comment.
```suggestion
  ATAN2,                 ///< 2-argument arctangent
```

Reply:
done